test/qa/lib/fileshare.py

```diff
@@ -52,6 +52,16 @@ def start_transfer(peer_address: str, *filepaths: str) -> sh.RunningCommand:
 
 def get_last_transfer(outgoing: bool = True, ssh_client: ssh.Ssh = None) -> Optional[str]:
     """Return last id of the last received or sent transfer."""
+    transfer_ids = get_all_transfers(outgoing, ssh_client)
+
+    if len(transfer_ids) == 0:
+        return None
+
+    return transfer_ids[-1]
+
+
+def get_all_transfers(outgoing: bool = True, ssh_client: ssh.Ssh = None) -> list[str]:
```

Comment:
Yes, you are right. I misread `outgoing` as `ongoing`. Fixing. 
I'll also find only the transfers which are not finished, otherwise we'll have a lot of noise in the logs  about canceling already finished transfer

Reply:
Done